core/src/text/textBuffer.cpp

```diff
@@ -15,6 +15,22 @@ TextBuffer::TextBuffer(std::shared_ptr<VertexLayout> _vertexLayout)
 TextBuffer::~TextBuffer() {
 }
 
+std::vector<TextBuffer::WordBreak> TextBuffer::findWordBreaks(const std::string& _text) {
```

Comment:
Looks like we also need to handle explicit line breaks (`\n` chars) in the text, to match JS (unless I missed it somewhere). For example, the string `One\nTwo\nThree` should render as:

```
One
Two
Three
```